texastribune/donations
# Add updated queso classes

#### What's this PR do?
Updates naming of queso classes to match v5

#### Why are we doing this? How does it help us?

See https://github.com/texastribune/texastribune/pull/3662 and  https://github.com/texastribune/texastribune/pull/3662

#### How should this be manually tested?

`yarn` to get the latest and `yarn run build` to compile. Then just peek at the portal and make sure everything looks as it was before.

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?
Shouldn't be

#### What are the relevant tickets?
Part of larger refactor.

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
